packages/hidp/hidp/federated/models.py

```diff
@@ -53,6 +53,10 @@ class OpenIdConnection(models.Model):
     # Unique identifier for the user at the identity provider (i.e. the "sub" claim)
     # Guaranteed to be unique together with the issuer_claim.
     subject_claim = models.CharField(max_length=255)
+    # The last time the user logged in using this OpenID connection.
+    last_login_at = models.DateTimeField(
+        _("last login using this connection"), null=True, blank=True
+    )
```

Comment:
See other discussion thread

```suggestion
    # The last time the user used this OpenID connection.
    last_usage = models.DateTimeField(
        _("last usage of this connection"), default=timezone.now
    )
```

Reply:
Changed!